crowdsourcing/models.py

```diff
@@ -298,6 +298,12 @@ class Task(models.Model):
 class TaskWorker(models.Model):
     task = models.ForeignKey(Task, related_name='task_workers')
     worker = models.ForeignKey(Worker)
+    statuses = ((1, 'In Progress'),
```

Comment:
because we want to accept a task rather than a taskworkerresult. it doesnt make sense to have to accept/reject each individual input item for a task (at least at first).